WordPress/src/main/res/values/strings.xml

```diff
@@ -2906,6 +2906,8 @@
     <string name="dialog_edit_story_unavailable_message">Unable to load media for this story. Check your internet connection and try again in a moment.</string>
     <string name="dialog_edit_story_unrecoverable_title">Can\'t edit Story</string>
     <string name="dialog_edit_story_unrecoverable_message">We couldn\'t find the media for this story on the site.</string>
+    <string name="dialog_edit_story_unsupported_format_title">Unsupported format</string>
+    <string name="dialog_edit_story_unsupported_format_message">Stories don\'t support GIF files at the moment; one or more slides have not been added to your Story. Please choose a static image or video background instead.</string>
```

Comment:
```suggestion
    <string name="dialog_edit_story_unsupported_format_message">One or more slides have not been added to your Story because Stories don\'t support GIF files at the moment. Please choose a static image or video background instead.</string>
```

I just moved this around a bit so that it tells the user what happened first and why the issue occurred.